modules/@apostrophecms/asset/index.js

```diff
@@ -85,9 +131,11 @@ module.exports = {
             }
           }
 
-          function buildPublicCssBundle() {
+          async function buildPublicCssBundle() {
             const publicImports = getImports('public', '*.css', { });
-            fs.writeFileSync(`${bundleDir}/public-bundle.css`,
+            await fs.ensureFile(`${bundleDir}/${PUBLIC_BUNDLE_CSS}`);
```

Comment:
This was for the test suite. For some reason when it ran without this it choked, even though `writeFileSync` should create the file. The same process is done for the JS bundle, but it isn't coking on that. Not sure why that is.

@boutell Any ideas?

Reply:
See my note earlier about the lack of await on a mkdirp call. May not have happened yet.